src/js/drawer.js

```diff
@@ -50,31 +51,38 @@
           options = $.extend({}, options);
           $this.data(namespace, { options: options });
 
-          var iScroll = new IScroll('.' + __.settings.class.nav, options.iscroll);
+          __.refresh.call(_this);
 
           if (options.showOverlay) {
             __.addOverlay.call(_this);
           }
 
           $('.' + __.settings.class.toggle).on('click.' + namespace, function() {
             __.toggle.call(_this);
-            return iScroll.refresh();
+            return _this.iScroll.refresh();
```

Comment:
Are you sure about this case, @blivesta? The behaviour for `_this.iScroll.refresh()` is different than `__.refresh.call(_this)`; they should have the same effect, but the `__.refresh.call(_this)` re-creates the whole `iScroll`, which should be slower than calling `_this.iScroll.refresh()`.

I actually added this piece of code after I made the PR, so this represents another need that I had: to re-create the `iScroll` after I dinamically changed the inner HTML for the drawer. I tried to minimize the changes (therefore just replacing `iScroll.refresh()` with `_this.iScroll.refresh()`), but maybe the name I chose wasn't appropriate? Do you think something more explicit like `recreateIScroll` would be better?